api/v6/report_server.thrift

```diff
@@ -13,6 +13,70 @@ namespace js codeCheckerDBAccess_v6
 const i64 MAX_QUERY_SIZE = 500
 //=================================================
 
+struct BugPathEvent {
+  1: i64    startLine,
+  2: i64    startCol,
+  3: i64    endLine,
+  4: i64    endCol,
+  5: string msg,
+  6: i64    fileId
+  7: string filePath
+}
+typedef list<BugPathEvent> BugPathEvents
+
+struct BugPathPos {
+  1: i64    startLine,
+  2: i64    startCol,
+  3: i64    endLine,
+  4: i64    endCol,
+  5: i64    fileId
+  6: string filePath
+}
+typedef list<BugPathPos> BugPath
+
+// TODO: This seems to be unused?
+struct SuppressBugData {
+  1: string bugHash,
+  2: string fileName,
+  3: string comment
+}
+typedef list<SuppressBugData> SuppressBugList
+
+struct ConfigValue {
+  1: string checkerName,
+  2: string attribute,
+  3: string value
+}
+typedef list<ConfigValue> CheckerConfigList
+
+enum Severity {
```

Comment:
For my eyes it would be better to put every `enum` definition at the top of this file after `const` declaration.

Reply:
Why do we need to specify values (0,10, etc.) for this enum?

Reply:
Sure, I'll reorganise this, just first merge the other prs editing this file.